### DISCUSSION TITLE
How to replace Windows's taskbar with yasb?

### DISCUSSION THREAD

**Discussion by DeadlySquad13** [amnweb/yasb#32] [Q&A] [answered]
Hello, thanks for your effort building this cool program!
I want to achieve something clean resembling screenshots in the README. How can I disable default taskbar?
![image](https://github.com/user-attachments/assets/8264777d-b241-4cbd-8225-9a18dab8e315)

I've used Window's setting to hide taskbar but unfortunately it pop ups from time to time because of it's emergency mechanism when, for example, a new notification appears: 
![image](https://github.com/user-attachments/assets/515bc4d8-81fc-4ad7-b7f9-619d4b02bb81)

The main reason I decided to try custom taskbars and yasb in particular was to get a taskbar replacement that won't distract me with these notifications - according to this help page it's not possible in default one https://answers.microsoft.com/en-us/windows/forum/all/stop-taskbar-from-not-autohiding-when-receiving/0096e543-d7db-46ce-8792-2cf9782c7212

Using Windows 10 but would be interested in Windows 11 solutions too.

Hope you'll help me find an answer!

**Comment by amnweb** [comment, marked as answer]
Did you try this https://github.com/LuisThiamNye/ButteryTaskbar2 ?

**Reply by DeadlySquad13** [reply]
Thanks! I will try it out!

UPD: Worked perfectly!
